tests/unit/api/test_files.py

```diff
@@ -61,3 +61,13 @@ async def test_list_paginated(self, files_api: FilesAPI, mocked_deepset_cloud_ap
             ],
             has_more=False,
         )
+        mocked_deepset_cloud_api.get.assert_called_once_with(
```

Comment:
Just added this assert that the api is correclty called under the hood.

Reply:
I don't think it matters for the test really, but I'm not sure if it is a valid example.

Reply:
I can create an integration test for this. Lets see

Reply:
Sorry, my earlier point here is that I'm not sure the filter works with `created_at` specifically. Again doesn't really matter for the test.